components/brave_new_tab_ui/containers/newTab/settings/backgroundImage.tsx

```diff
@@ -39,64 +65,101 @@ class BackgroundImageSettings extends React.PureComponent<Props, {}> {
+
     return (
-      <div>
-        <SettingsRow>
-          <SettingsText>{getLocale('showBackgroundImage')}</SettingsText>
-          <Toggle
-            onChange={toggleShowBackgroundImage}
-            checked={showBackgroundImage}
-            size='large'
-          />
-        </SettingsRow>
-        <SettingsRow isChildSetting={true}>
-          <SettingsText>{getLocale('brandedWallpaperOptIn')}</SettingsText>
-          <Toggle
-            onChange={toggleBrandedWallpaperOptIn}
-            // This option can only be enabled if
-            // users opt in for background images
-            checked={showBackgroundImage && brandedWallpaperOptIn}
-            disabled={!showBackgroundImage}
-            size='small'
+      <>
+        {this.state.location === Location.LIST && (
+          <div>
+            <SettingsRow>
+              <SettingsText>{getLocale('showBackgroundImage')}</SettingsText>
+              <Toggle
+                onChange={toggleShowBackgroundImage}
+                checked={showBackgroundImage}
+                size='large'
+              />
+            </SettingsRow>
+            <SettingsRow isChildSetting={true}>
+              <SettingsText>{getLocale('brandedWallpaperOptIn')}</SettingsText>
+              <Toggle
+                onChange={toggleBrandedWallpaperOptIn}
+                // This option can only be enabled if
+                // users opt in for background images
+                checked={showBackgroundImage && brandedWallpaperOptIn}
+                disabled={!showBackgroundImage}
+                size='small'
+              />
+            </SettingsRow>
+            {showBackgroundImage && featureCustomBackgroundEnabled && (
+              <StyledCustomBackgroundSettings>
+                <StyledCustomBackgroundOption
+                  onClick={this.onClickCustomBackground}
+                >
+                  <StyledSelectionBorder selected={usingCustomBackground}>
+                    <StyledUploadIconContainer selected={usingCustomBackground}>
+                      <UploadIcon />
+                      <StyledUploadLabel>
+                        {getLocale('customBackgroundImageOptionUploadLabel')}
+                      </StyledUploadLabel>
+                    </StyledUploadIconContainer>
+                  </StyledSelectionBorder>
+                  <StyledCustomBackgroundOptionLabel>
+                    {getLocale('customBackgroundImageOptionTitle')}
+                  </StyledCustomBackgroundOptionLabel>
+                </StyledCustomBackgroundOption>
+                <StyledCustomBackgroundOption
+                  onClick={this.onClickBraveBackground}
+                >
+                  <StyledSelectionBorder selected={usingBraveBackground}>
+                    <StyledCustomBackgroundOptionImage style={{ backgroundImage: `url(${braveBackground})` }} selected={usingBraveBackground}/>
```

Comment:
Do we need to explicitly set `style` here? Can we not pass `braveBackground` into the styled component anymore?